config/openiotsdk/mbedtls/mbedtls_config.h

```diff
@@ -24,9 +24,6 @@
  *  limitations under the License.
  */
 
-#ifndef MBEDTLS_CONFIG_H
-#define MBEDTLS_CONFIG_H
```

Comment:
Why do we remove this?

Reply:
Ok, I saw the migration guide. I get an answer to my question. However, what do you think about adapting this file at all? I mean copy `mbedtls_config.h` from mbedtls sources and comment/uncomment necessary settings?

Reply:
It is required as part of the mbedtls migration from v2 to v3

Reply:
We could update our local version to the upstream mbedtls_config.h it might make any ongoing maintenance easier that's for sure. I'll pull it in and tweak the settings we need and again try the psa building.

Reply:
What do you mean by upstream mbedtls_config.h?

Reply:
The one supplied as part of mbedtls 3.2.1, modified for any specific settings we require